src/Annotations/AppKernel.php

```diff
@@ -109,10 +109,10 @@ public function __construct(string $environment = 'dev')
         $this->app->register(new ApiProblemProvider());
         $this->app->register(new ContentNegotiationProvider());
         $this->app->register(new PingControllerProvider());
+        $this->app->register(new ServiceControllerServiceProvider());
 
         if ($this->app['debug']) {
             $this->app->register(new HttpFragmentServiceProvider());
```

Comment:
probably the other two providers here are not necessary, as they deal with templating and this service only returns JSON responses; but it's a separate simplification